plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTable.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.testing.sql.SqlExecutor;
+
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.function.Function;
+
+import static com.google.common.base.Verify.verify;
+import static java.lang.Character.MAX_RADIX;
+import static java.lang.Math.abs;
+import static java.lang.Math.min;
+import static java.lang.String.format;
+import static java.lang.String.join;
+import static java.util.stream.Collectors.joining;
+
+public class TestCassandraTable
+        implements AutoCloseable
+{
+    private static final SecureRandom random = new SecureRandom();
+    // The suffix needs to be long enough to "prevent" collisions in practice. The length of 5 was proven not to be long enough
+    private static final int RANDOM_SUFFIX_LENGTH = 10;
+
+    private final SqlExecutor sqlExecutor;
+    private final String keyspace;
+    private final String tableName;
+
+    // Make sure the `namePrefix` in the following constructors does not exceed 38 characters,
+    // because Cassandra table names are limited to 48 characters in size, so we can only use
+    // a maximum of 38 characters without the suffix.
+    // https://cassandra.apache.org/doc/trunk/cassandra/cql/ddl.html#common-definitions
+    //
+    // NOTE: "38 characters" may be changed in the future, please refer to
+    // `48 - RANDOM_SUFFIX_LENGTH` for the exact value.
+
+    public TestCassandraTable(
+            SqlExecutor sqlExecutor,
+            CassandraServer server,
+            String keyspace,
+            String namePrefix,
+            List<ColumnDefinition> columnDefinitions,
+            List<String> rowsToInsert)
+    {
+        this.sqlExecutor = sqlExecutor;
+        this.keyspace = keyspace;
+        this.tableName = namePrefix + randomTableSuffix();
+        sqlExecutor.execute(format("CREATE TABLE %s.%s %s", keyspace, tableName, tableDefinition(columnDefinitions)));
+        try {
+            String columns = columnDefinitions.stream()
+                    .map(columnDefinition -> columnDefinition.name)
+                    .collect(joining(", "));
+            for (String row : rowsToInsert) {
+                // some databases do not support multi value insert statement
```

Comment:
Remove unhelpful comment or rephrase as "Cassandra does not support..."

Reply:
I have confirmed that Cassandra does not support multi value insert statement, so I'll use the description "Cassandra does not support..."